packages/zora/src/Zora.ts

```diff
@@ -0,0 +1,46 @@
+import {
+  type TransactionFilter,
+  type MintActionParams,
+  compressJson,
+} from '@rabbitholegg/questdk'
+import { type Address } from 'viem'
+import { CHAIN_ID_ARRAY } from './chain-ids'
+import { ZORA_MINTER_ABI } from './abi'
+
+export const mint = async (
+  mint: MintActionParams,
+): Promise<TransactionFilter> => {
+  const { chainId, contractAddress, tokenId, amount, recipient } = mint
+
+  let andArray = []
```

Comment:
The linter wants this to be a const, since it does not get reassigned. Not really a big deal though as it still passes the tests.